extension/src/fetch.ts

```diff
@@ -38,7 +38,8 @@ export async function fetchAndUpdate(ctx: ExtensionContext, prompt = true) {
   if (shouldUpdate) {
     const config = workspace.getConfiguration()
     config.update('explorer.experimental.fileNesting.enabled', true, true)
-    config.update('explorer.experimental.fileNesting.expand', false, true)
+    config.update('explorer.experimental.fileNesting.expand',
+      config.inspect('explorer.experimental.fileNesting.expand').globalValue ?? false, true)
```

Comment:
Maybe we could use a `if` so we don't touch it when the user set it explicitly?

Reply:
Sure, that is updated :+1: